addons/cookoff/stringtable.xml

```diff
@@ -16,38 +16,14 @@
             <Portuguese>ACE Cook off</Portuguese>
             <Czech>ACE Vznícení munice</Czech>
         </Key>
-        <Key ID="STR_ACE_CookOff_enable_hd_name">
-            <English>Damage handling and turret effects</English>
-            <Spanish>Daño y efectos de torreta</Spanish>
-            <German>Schadensberechnung und Geschützturmeffekte</German>
-            <Japanese>損傷処理と砲塔の効果</Japanese>
-            <Russian>Обработка урона и эффектов срыва башни</Russian>
-            <Portuguese>Manipulação de dano e efeitos de torre</Portuguese>
-            <French>Dégâts et effets de tourelle</French>
-            <Chinese>傷害控制及炮塔效果</Chinese>
-            <Chinesesimp>损坏处理和炮塔效果</Chinesesimp>
-            <Italian>Gestione danni ed effetti torretta</Italian>
-            <Czech>Poškodit ovládání a efekty věže</Czech>
-            <Polish>Obsługa obrażeń i efekty wieży</Polish>
-            <Korean>피해량 조절 및 포탑에 효과 부여</Korean>
-        </Key>
-        <Key ID="STR_ACE_CookOff_enable_hd_tooltip">
-            <English>Changes damage handling for cook off and turret explosion effects</English>
-            <Spanish>Cambia el daño de la detonación inducida por calor y los efectos de la explosión de la torreta</Spanish>
-            <German>Ändert die Schadensberechnung für die Durchzündung und die Explosionseffekte des Geschützturmes</German>
-            <Japanese>誘爆の損傷処理と砲塔の爆発効果を変更します。</Japanese>
-            <Russian>Изменяет обработку урона для возгорания и эффекта срыва башни</Russian>
-            <Portuguese>Modifica a manipulação de dano para o cozinhamento de munição e efeitos de explosão da torre</Portuguese>
-            <French>Modifie la gestion des dégâts pour l'auto-inflammation et les effets d'explosion de tourelle.</French>
-            <Chinese>更改殉爆以及炮塔爆炸之傷害控制</Chinese>
-            <Chinesesimp>改变殉爆和炮塔爆炸的损坏处理效果</Chinesesimp>
-            <Italian>Modifica la gestione dei danni per l'esplosione di munizioni e gli effetti di esplosione della torretta</Italian>
-            <Czech>Změní poškození ovládání a efekty výbuchu veže</Czech>
-            <Polish>Zmienia obsługę obrażeń podczas samozapłonu i eksplozji wieży</Polish>
-            <Korean>쿡오프로 인해 피해량의 변화와 포탑 터짐현상을 결정합니다.</Korean>
+        <Key ID="STR_ACE_CookOff_enable_name">
+            <English>Enable vehicle cook-off</English>
```

Comment:
Are you sure about this?

My changes as is will already have some backwards compatibility 'breaking' effects, specifically: `GVAR(enable)` previously affected a mix of fire effects and ammo detonation. However, given the editor settings, this to me did not make sense, so I changed so that `GVAR(enable)` affects fire effects only.

If you still want a name change (which tbh I'm all for), I'd recommend `GVAR(enableFire)` instead, as that's an already existing setting (although one could use that as a very solid counter argument).

A problem will remain though:
https://github.com/acemod/ACE3/blob/master/docs/wiki/framework/cookoff-framework.md#1-disabling--enabling-cook-off-for-individual-vehicles
`GVAR(enable)` set on objects is API, so we shouldn't (but could if really wanted) change that.

Reply:
Best to keep backwards compat then, but setting name should be more specific IMO.

Reply:
Updated.

Reply:
[cc06b37](https://github.com/acemod/ACE3/pull/9758/commits/cc06b37f47adee85c080f55c0c2740a1da1a3e9c) removes `GVAR(enable)` in favor of `GVAR(enableFire)`, which should make it even more clear.